universum/initializer.py

```diff
@@ -0,0 +1,51 @@
+from pathlib import Path
+
+from .modules.output.output import MinimalOut
+from .lib.gravity import Module
+from .lib.module_arguments import ModuleArgumentParser
+
+__all__ = ["Initializer"]
+
+
+class Initializer(Module):
```

Comment:
I think "Initializer" is too abstract name.
Please change to "ProjectInitializer" or even "ConfigsPyCreator".